how-to-build.md

```diff
@@ -2,10 +2,10 @@ This file explains how to build and install the Realm core library.
 
 ## Prerequisites
 
-To build the Realm core library, you need the standard set of build
-tools. This includes a C/C++ compiler and GNU make. Realm is
-thoroughly tested with both GCC and Clang. It is known to work with
-GCC 4.2 and newer, as well as with Clang 3.0 and newer.
+To build the Realm core library, you need cmake and a standard set of
+build tools. This includes a C/C++ compiler and a build system like
+GNU make. Realm is thoroughly tested with both GCC and Clang. It is
+known to work with GCC 4.2 and newer, as well as with Clang 3.0 and newer.
```

Comment:
Yes, I believe we require GCC >= 4.9, because it is the earliest version that has sufficient C++14 support.